### PR TITLE
Ref task #2830 Apply CV

### PR DESCRIPTION
**Pull request by buingocmay**
image

### CODE REVIEW

**Comment by quang5566** [comment]
LGTM

**Comment by buingocmay** [comment]
ready

**Comment by DuongNT1512** [comment]
tự kiểm tra lỗi indent

**Comment by DuongNT1512** [comment]
commented

**Comment by buingocmay** [comment]
ready

File: app/views/jobs/_apply.html.erb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<%= form_with model: @user.user_curriculum_vitaes.build, local: true do |f| %>
```
Comment (DuongNT1512) [inline app/views/jobs/_apply.html.erb]: build trong controller nhé em

File: app/views/jobs/show.html.erb [hunk 2]
```diff
@@ -53,7 +53,43 @@
           <%= link_to t(".delete"), job_path, method: :delete,
             data: {confirm: t(".sure")}, class: "btn btn-lg btn-danger" %>
         <% end %>
+        <% if !logged_in? || current_user.candidate? %>
+          <%= link_to t(".apply"), new_user_curriculum_vitae_path, class:
+            "btn wthree-bnr-btn text-capitalize", "data-toggle" => "modal",
+            "data-target" => "#applyCV" %>
```
Comment (DuongNT1512) [inline app/views/jobs/show.html.erb]: convention

**Comment by DuongNT1512** [comment]
commented

**Comment by buingocmay** [comment]
ready

**Comment by thanhhungchu95** [comment]
merged

**Pull request merged**